packages/shared/src/telemetry/events/method-called.ts

```diff
@@ -1,6 +1,8 @@
+import { TelemetryClientCache } from '../clientCache';
 import type { TelemetryEventRaw } from '../types';
 
 const EVENT_METHOD_CALLED = 'METHOD_CALLED' as const;
+const EVENT_SAMPLING_RATE = 0.1;
```

Comment:
Also introducing a sampling rate lower than the default for this event, to handle the server-side scenarios.